tekton-pipelines/images/art-tools-tester/Containerfile

```diff
@@ -0,0 +1,53 @@
+FROM registry.access.redhat.com/ubi8/ubi:8.5 AS builder
+
+# Install build dependencies
+WORKDIR /usr/local/src
+USER root
+RUN dnf -y module enable python36 \
+  && dnf -y install python3 python3-wheel python3-devel gcc krb5-devel wget tar gzip \
+  && python3 -m pip install "pip >= 21"
+
+# Download oc
+ARG OC_VERSION=latest
+RUN wget -O openshift-client-linux-"$OC_VERSION".tar.gz https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/"$OC_VERSION"/openshift-client-linux.tar.gz \
+  && tar -xzvf openshift-client-linux-"$OC_VERSION".tar.gz oc kubectl
+
+# Download Tekton CLI
+ARG TKN_VERSION=0.23.1
+RUN wget -O "tkn_${TKN_VERSION}_Linux_x86_64.tar.gz" https://github.com/tektoncd/cli/releases/download/v0.23.1/"tkn_${TKN_VERSION}_Linux_x86_64.tar.gz" \
+  && tar -xzvf ./tkn_0.23.1_Linux_x86_64.tar.gz tkn
```

Comment:
`TKN_VERSION` here as well.
Any reason we're not pulling from http://mirror.openshift.com/pub/openshift-v4/x86_64/clients/pipeline/?